src/momento/utilities/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .expiration import Expiration, ExpiresAt, ExpiresIn
+from .shared_sync_asyncio import DEFAULT_EAGER_CONNECTION_TIMEOUT_SECONDS, str_to_bytes
```

Comment:
You'll also need an [`__all__` definition](https://docs.python.org/3/tutorial/modules.html#importing-from-a-package) with the names here